opa/opa_test.go

```diff
@@ -161,7 +165,7 @@ var _ = Describe("opa client", func() {
 						createPolicyResponse = httpmock.NewStringResponse(http.StatusInternalServerError, "OPA error")
 					})
 
-					It("should return a publish policy bad response error", func() {
+					XIt("should return a publish policy bad response error", func() {
```

Comment:
are these tests meant to be skipped?

Reply:
yeah, these are meant to test the path when it needs to pull a policy from elasticsearch and push it to OPA which hasn't been implemented yet.